project_resource_calendar/__manifest__.py

```diff
@@ -19,12 +19,12 @@
         'python': [],
     },
     'data': [
-        'views/room_type.xml',
-        'views/equipment_calendar.xml',
-        'views/resource.xml',
-        'views/sector.xml',
-        'views/room.xml',
-        'views/instrument.xml',
+        'views/room_type_view.xml',
+        'views/equipment_calendar_view.xml',
+        'views/resource_view.xml',
+        'views/sector_view.xml',
+        'views/room_view.xml',
+        'views/instrument_view.xml',
```

Comment:
@eilst: One one extra line at the end of the file [As per the coding standard and guidelines]